src/spatio_temporal_voxel_grid.cpp

```diff
@@ -330,12 +341,38 @@ double SpatioTemporalVoxelGrid::GetFrustumAcceleration( \
 }
 
 /*****************************************************************************/
-void SpatioTemporalVoxelGrid::GetOccupancyPointCloud( \
-                                       pcl::PointCloud<pcl::PointXYZ>::Ptr& pc)
+void SpatioTemporalVoxelGrid::GetOccupancyPointCloud(\
+                                       sensor_msgs::PointCloud2::Ptr& pc2)
 /*****************************************************************************/
 {
-  // return the pointcloud stored
-  pc = _pc;
+  // convert the grid points stored in a PointCloud2
+  sensor_msgs::PointCloud2::Ptr grid_points_cloud (new sensor_msgs::PointCloud2());
```

Comment:
You have this `typedef`ed, if you use it'll be less than max line length ;)

Reply:
Where? I corrected the line length anyway

Reply:
Not relevant anymore since we use the input pointer